MANIFEST.in

```diff
@@ -9,8 +9,6 @@ include HOWTORELEASE.rst
 include tox.ini
 include setup.py
 
-include .coveragerc
-
 include plugin-test.sh
 include requirements-docs.txt
```

Comment:
Also, I wonder if we should remove those scripts as well? They all seem to be part of a release/testing workflow which is no longer true:

* `plugin-test.sh`: test a bunch of plugins located in parent directories using current pytest;
* `requirements-docs.txt`: we should advertise using `tox -e docs` instead.
* `runtox.py`: runs `tox` while pointing to @hpk42 index... it probably made sense when `py` and `pytest` release were really tied together, but now I don't think it makes sense.